tests/CMake/utils.cmake

```diff
@@ -0,0 +1,123 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (C) 2022 Xilinx, Inc. All rights reserved.
+#
+
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED OFF)
+set(CMAKE_VERBOSE_MAKEFILE ON)
+set(XILINX_XRT $ENV{XILINX_XRT})
+set(XRT_CORE_LIBRARY xrt_core)
+set(XILINX_VITIS $ENV{XILINX_VITIS})
+
+if(NOT VITIS_COMPILER)
+  find_program(VITIS_COMPILER 
+  NAMES v++
+  HINTS XILINX_VITIS}
+  )
+  if(VITIS_COMPILER)
+    message("Vitis Compiler is found at ${VITIS_COMPILER}")
+  else()
+   message(FATAL_ERROR "Vitis Compiler v++ is not found, Please source vitis and try again")
```

Comment:
No finding Vitis should not be a fatal error.   It should at most be a warning saying that xclbin cannot be built without a Vitis installation available

Reply:
Hi @stsoe ,
A warning might lead to confusion . So the message (if v++ not found ) is removed all together.